defaults/defaults.go

```diff
@@ -39,6 +39,8 @@ const (
 
 	HubbleSocketPath       = "/var/run/cilium/hubble.sock"
 	HubbleServerSecretName = "hubble-server-certs"
+  	HubbleServerSecretCertName = "tls.crt"
```

Comment:
Why are these changes necessary? :thinking:

Reply:
There was conflict with master, so I fixed it.